### PR TITLE
Modifiers - expose extractAccessibilityProps & extractComponentProps

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
Modifiers - expose extractAccessibilityProps & extractComponentProps

## Changelog
Modifiers - expose extractAccessibilityProps & extractComponentProps

### CODE REVIEW

File: src/commons/index.js [hunk 1]
```diff
@@ -27,5 +27,11 @@ module.exports = {
   },
   get withScrollReached() {
     return require('./withScrollReached').default;
+  },
```
Comment (ethanshar) [inline src/commons/index.js:30]: Let's extract the whole Modifiers file..

Comment (M-i-k-e-l) [reply src/commons/index.js:30]: Fixed

File: src/index.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ export {
 } from './commons/new';
 export * as Incubator from './incubator';
 export * as Hooks from './hooks';
+export * as Modifiers from './commons/modifiers';
```
Comment (ethanshar) [inline src/index.ts:21]: I think this is incorrect. 
This will export all Modifiers functions separately, while we want to export it as a whole class.

**Review by ethanshar — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 2 above)
Comment (M-i-k-e-l) [reply src/index.ts:21]: You can see the usage in the private lib's PR, it is as follows:
```
import {Modifiers} from 'react-native-ui-lib';
{...Modifiers.extractAccessibilityProps(this.props)}
```
Is that not what you meant?

Comment (ethanshar) [reply src/index.ts:21]: yes.. it's actually correct. 
Not sure why it look weird before :/
merging..

**Review by ethanshar — APPROVED** [review]

**Pull request merged**